### ISSUE TITLE
Re-merge txtlsim repositories

### ISSUE THREAD

**Issue by murrayrm** [vipulsinghal02/txtlsim_buildacell#15]
It looks like there are now two independent repositories for this project: one at https://github.com/BuildACell/txtlsim and one here.  I believe the one on the BuildACell project is the "master" repository, but somehow this repository is not forked from there.

We should get everything back to having one location for this toolbox.  I recommend either BuildACell or [biocircuits](https://github.com/biocircuits).

**Comment by vipulsinghal02** [comment]
Agreed. I think Zoltan and I have a few changes to make to the repo (examples / cleanup) and after that we can port this over to BuildACell, replacing the old one.

**Comment by vipulsinghal02** [comment]
PS: For now I think I will use the issue tracker in the BuildACell repo to guide the cleanup of this one too. I suspect some of those issues are valuable here too.
